sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java

```diff
@@ -228,13 +228,11 @@ public static synchronized FlitchioController getInstance(Context context) {
     /**
      * Check the installation of Flitchio Manager by comparing the version codes.
      *
-     * @throws FlitchioManagerDependencyException
+     * @return True if the Manager installed on the phone can be used by this version of the SDK.
      */
-    private static void checkFlitchioInstall(Context context)
-            throws FlitchioManagerDependencyException {
-        if (!BuildConfig.DEBUG && getFlitchioManagerVersionCode(context) < getVersionCode()) {
-            throw new FlitchioManagerDependencyException();
-        }
+    private static boolean isFlitchioManagerUsable(Context context) {
+        return BuildConfig.DEBUG // In debug, we consider the Manager always accessible
```

Comment:
This is legacy code from when the SDK was used directly by the Manager (not through Gradle). I'll remove it now but we might need to reconsider it when we have internal builds (even though everything should be fine).